contracts/test/PoolSwapTest.sol

```diff
@@ -45,6 +44,14 @@ contract PoolSwapTest is ILockCallback {
         if (ethBalance > 0) CurrencyLibrary.NATIVE.transfer(msg.sender, ethBalance);
     }
 
+    function _mintForSelf(Currency currency, uint256 amount) internal {
+        manager.mint(currency, address(this), amount);
+    }
+
+    function _burnFromSelf(Currency currency, uint256 amount) internal {
+        manager.burn(currency, amount);
+    }
```

Comment:
do these actually help readability? think just inlining the calls is better

Reply:
Yea good point before I had some other logic that I removed